prometheus_client/core.py

```diff
@@ -329,6 +347,19 @@ def _samples(self):
 
 @_MetricWrapper
 class Summary(object):
+    '''
+    A Summary tracks the size and number of events.
+
+    Example use cases for Summaries:
+    - Response latency
+    - Request size
+
+    Example for a Summary:
+
+    from prometheus_client import Summary
+    s = Summary('request_latency_seconds', 'Description of summary')
+    s.observe(4.7)    # Observe 4.7 (seconds in this case)
```

Comment:
`time()` would be a better way to demonstrate tracking durations, this example would encourage users to do that by hand. I'd suggest having a time() example, and a separate one for something like bytes